common/src/warp_runner/manager/commands/constellation_commands.rs

```diff
@@ -114,7 +123,114 @@ pub async fn handle_constellation_cmd(cmd: ConstellationCmd, warp_storage: &mut
             let r = rename_item(old_name, new_name, warp_storage).await;
             let _ = rsp.send(r);
         }
+        ConstellationCmd::DeleteItems { item, rsp } => {
+            let r = delete_items(warp_storage, item).await;
+            let _ = rsp.send(r);
+        }
+    }
+}
+
+async fn delete_items(
```

Comment:
i tested this function on Linux and it seems to work. much simpler:
```
fn delete_items(
    warp_storage: &mut warp_storage,
    item: Item,
) -> Result<uplink_storage, Error> {
    let cwd = warp_storage.current_directory()?;
    cwd.remove_item(&item.name())?;
    get_items_from_current_directory(warp_storage)
}
```

can we use this instead? requesting changes here but will approve if for some reason this doesn't work on a different platform.